man/bootctl.xml

```diff
@@ -183,6 +183,12 @@
         <listitem><para>Do not touch the firmware's boot loader list stored in EFI variables.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--graceful</option></term>
```

Comment:
Hmm, maybe `--permissive` or `--sloppy` rather than `--graceful`? I understand where you're coming from, but if one doesn't think about "graceful failure", then this is name is very unclear.

Reply:
I think we can still change the name before the final release. Let's merge this as is now.